### DISCUSSION TITLE
self hosted - multiple domain login RuntimeSessionAdapter

### DISCUSSION THREAD

**Discussion by tallesairan** [plausible/analytics#3164] [Self-Hosted Support]
I use a single instance to log in to different subdomains to better organize the sites, but now it has stopped working, it only accesses from the main domain set in the plausible env, would you have any idea how to proceed with this, I am trying to change it in the RuntimeSessionAdapter but without success so far, does anyone have any idea how to get this working again?

**Comment by ruslandoga** [comment]
@tallesairan 👋

Has anything changed in your setup when it stopped working? Does anything show up in the `docker compose logs plausible` when you try accessing through a subdomain?

**Reply by tallesairan** [reply]
Sorry for the delay but I managed to solve it, it was a commit that had to make a change by adding a handler to the session but I managed to fix it for my use
i just removed the `plug PlausibleWeb.Plugs.RuntimeSessionAdapter, @session_options`
and replaced by `plug Plug.Session, @session_options `
